contracts/token/mocks/ERC20Mock.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+/* solhint-disable */
+pragma solidity 0.8.26;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+
+contract ERC20Mock is ERC20 {
```

Comment:
small nitpick, but `MockERC20` is clearer and consistent with the way we always name it